environment.py

```diff
@@ -372,6 +372,14 @@ def env() -> Mapping[str, Optional[str]]:
         # Typically only enabled on main deployments.
         'AZUL_ENABLE_MONITORING': '0',
 
+        # alarm config = number of evaluation periods, 'x',
+        #                length of each evaluation period in seconds, 's>',
+        #                threshold
+        # If ``AZUL_ENABLE_MONITORING`` is set to 1, alarms will activate if the
+        # number of 5xx errors for a given component (e.g. service or indexer)
+        # exceeds the specified threshold over the specified evaluation periods.
+        'AZUL_ALARM_CONFIG': '6x600s>10',
```

Comment:
```suggestion
        'azul_alarm_config': '6x600s>10',
```

because there is no point in exporting it to the lambda environment.